github/ci/prow/files/jobs/kubevirt/cluster-network-addons-operator/cluster-network-addons-operator-presubmits.yaml

```diff
@@ -258,7 +258,7 @@ presubmits:
         - release-\d+\.\d+
       annotations:
         fork-per-release: "true"
-      always_run: false
+      always_run: true
       optional: true
```

Comment:
If you feel comfortable that they pass, can we make them mandatory?

Reply:
yes, I'll open a separate PR for that